examples/rockpaperscissors/python_example.py

```diff
@@ -11,38 +11,74 @@ class ConvNet(nn.Module):
     Convolutional Neural Network to classify hand gestures in an image as rock, paper or scissors
     """
 
-    def __init__(self):
+    def __init__(self, in_channels=3, num_features=3):
```

Comment:
This file shouldn't be changed as it's an example file.

Reply:
Anyways having a slight variation , I think it won't affect the generality , will it ?

Reply:
No, it won't, but changing a working and clear code is basically redundant.

Reply:
Okay I will revert these changes